088.Merge Sorted Array/088.Merge Sorted Array.md

```diff
@@ -0,0 +1,33 @@
+# 088.Merge Sorted Array
+
+**<font color=red>level: Easy</font>**
+
+##  Description
+
+Given two sorted integer arrays A and B, merge B into A as one sorted array.
+Note: You may assume that A has enough space to hold additional elements from B. The number of elements initialized in A and B are m and n respectively.
+
+## Solution
+
+* Time complexity O(m+n)
+* Space complexity O(1)
+
+```c++
+class Solution{
+public:
+	void merge(int A[],int m,int B[], int n){
+		int la=m-1, lb=n-1, icur=m+n-1;
```

Comment:
1. 
int la = m - 1;
int lb = n - 1;
int icur= m + n - 1;
2. any variables should be meaningful.

Reply:
Very thanks for your careful watching!  I have corrected the formation and some mistakes. Because of the  change of No33. problem  in the leetcode.com, I will try to solve the newest problem.